### PR TITLE
"New flashy benchmark for comparing vanilla/native JS to ember and glimmer components:"

### PR DESCRIPTION
**Pull request by hergaiety**
Writes up "New flashy benchmark for comparing vanilla/native JS to ember and glimmer components:"
https://twitter.com/nullvoxpopuli/status/1335616011744251907

I only took a guess at the image path based on some other things I see in this repo, but I'm not confident it'll work. Can someone confirm?

### CODE REVIEW

File: source/2020-12-18-the-ember-times-issue-170.md [hunk 1]
```diff
@@ -1,27 +1,30 @@
...
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+> Using @glimmer/component instead of @ember/component can get you up to 2x speed boosts at almost half the memory consumption.
+> [@nullvoxpopuli](https://twitter.com/nullvoxpopuli) ([source](https://twitter.com/nullvoxpopuli/status/1335616011744251907))
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+Play with [the interactive browser demo](https://nullvoxpopuli.github.io/ember-three-boxes-demo/) or [explore the source](https://github.com/NullVoxPopuli/ember-three-boxes-demo) behind a performance comparison of Native JS, Ember Components and Glimmer Components. Feel the performance improvements the core team has achieved for yourself! 🔥
+
+![Glimmer rendering thousands of three.js elements with fast performance](/images/blog/emberjstimes/nullvoxpopuli-glimmer-performance.png)
```
Comment (amyrlam) [inline source/2020-12-18-the-ember-times-issue-170.md:25]: This should work! If we need to tweak the placement / alignment or something, we can always change it to HTML upstream, something like this: 

```<div class="float-right padded portrait-frame">Serena Fritsch</div>```

File: source/2020-12-18-the-ember-times-issue-170.md [hunk 2]
```diff
@@ -1,27 +1,30 @@
...
 
+Explore Ember/Glimmer component performance 🚅, .
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
 
 READMORE
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Glimmer and Ember component performance playground 🚅](https://twitter.com/nullvoxpopuli/status/1335616011744251907)
```
Comment (amyrlam) [inline source/2020-12-18-the-ember-times-issue-170.md]:
```suggestion
## [Glimmer and Ember component performance playground 🚅](https://nullvoxpopuli.github.io/ember-three-boxes-demo/)
```

**Review by amyrlam — APPROVED** [review]
This is awesome, thank you! 🙇‍♀️ 

Can you add your name to the bottom where it says `the crowd`?

**Comment by hergaiety** [comment]
@amyrlam suggestions addressed :)

**Comment by amyrlam** [comment]
@hergaiety thanks! 💯

**Pull request merged**